requirements-test.txt

```diff
@@ -0,0 +1,6 @@
+# Constraints on requirements below
```

Comment:
I'm not sure if the name or purpose of this file is clear enough, who is supposed to know they should use it?

Reply:
Do you mean there should be more information here on in the actual constraints file?

Reply:
Howzat?

Reply:
So, having had a think and done a bit of a research, I've decided to keep the filename as `requirements-test.txt` is quite a common name for this file and also it matches the pluralisation of the directory for the unittests (which is also a common name for that). I will try and improve the flow of the comments though.